packages/common/src/intl/locales/en/distribution.json

```diff
@@ -48,6 +50,9 @@
   "label.supply-quantity": "Quantity to supply",
   "label.units-in-pack-size-of_one": "unit in pack size of",
   "label.units-in-pack-size-of_other": "units in pack size of",
+  "messages.edit-service-charges": "Edit service charges",
+  "messages.stock-charges-description": "Total charges for lines related to stock. The tax amount is an effective tax rate calculated by the total tax paid over the subtotal of all stock lines.",
+  "messages.service-charges-description": "Total amounts of service charges. The tax amount is an effective tax rate calculated by the total tax paid over the subtotal of all service charges.",
```

Comment:
OK I seem to need a lot of help with this feature!

Tooltips I think are helpful. This sentence I think is not!